src/Flex.php

```diff
@@ -544,10 +556,12 @@ function ($value) {
             }
         }
 
-        foreach ($postInstallRecipes as $recipe) {
-            $this->configurator->postInstall($recipe, $this->lock, [
-                'force' => $event instanceof UpdateEvent && $event->force(),
-            ]);
+        if (method_exists($this->configurator, 'postInstall')) {
```

Comment:
Does this means that post-install configurators are not run when upgrading?
If yes, what about moving the postInstall method to a new class, so that we can load it even when the old configurator class is already loaded?

Reply:
We would have such issue any time we _edit_ a class of the plugin (except the main plugin class): when Flex is updated, it is possible that the loaded class is the old one rather than the new one.